app/assets/javascripts/redux/modules/models/listenSignals.js

```diff
@@ -101,21 +120,37 @@ export const reducer = handleActions({
 
   [LISTEN_SIGNAL_SHOW_REQUEST_SUCCESS]: handlesListenSignalSucccesResponse,
 
-  [LISTEN_SIGNAL_SHOW_REQUEST_FAIL]: (state, action) => ({
-    ...state,
-    data: {
-      ..._.get(state, 'data', {}),
-      [action.meta.id]: {
-        ..._.get(state, `data.${action.meta.id}`, {}),
-        error: action.payload,
-        loading: false,
-        loaded: false,
-      },
-    },
-  }),
+  [LISTEN_SIGNAL_SHOW_REQUEST_FAIL]: handlesListenSignalFailResponse,
+
+  [LISTEN_SIGNALS_POST_REQUEST]: handleSignalRequest,
 
   [LISTEN_SIGNALS_POST_REQUEST_SUCCESS]: handlesListenSignalSucccesResponse,
 
+  [LISTEN_SIGNALS_POST_REQUEST_FAIL]: handlesListenSignalFailResponse,
```

Comment:
This doesn't make sense since we don't have a signal id that we'll be passing in as a meta prop. This should be handled differently. We discussed how this can handled by a `flashMessage` reducer (maybe under `redux/modules/app.js`) that displays error and success messaging in the app.